pandas/tests/groupby/test_groupby.py

```diff
@@ -3688,6 +3688,18 @@ def test_nunique_with_timegrouper(self):
         )['data'].apply(pd.Series.nunique)
         tm.assert_series_equal(result, expected)
 
+    def test_nunique_with_timegrouper_and_nat(self):
+        test = pd.DataFrame({
```

Comment:
add the github issue number as a comment.

move this test to test_timegrouper.py (same dir)